modules/queue/workerpool.go

```diff
@@ -484,7 +484,7 @@ func (p *WorkerPool) doWork(ctx context.Context) {
 		case <-paused:
 			log.Trace("Worker for Queue %d Pausing", p.qid)
 			if len(data) > 0 {
-				log.Trace("Handling: %d data, %v", len(data), data)
+				log.Trace("Queue[%d] Handling: %d data, %v", p.qid, len(data), data)
```

Comment:
Yes. Otherwise you don't know what queue is handling the data.

Reply:
I mean are there any other string could be used here?

Reply:
nope - we're in the WorkerPool which doesn't have the name.

I think it would be possible to push the name up to WorkerPool but it's not there currently. I can't quite remember why I didn't have it up there in the first place.